### DISCUSSION TITLE
How can I create a pre-training model?

### DISCUSSION THREAD

**Discussion by tuoxiansp** [voicepaw/so-vits-svc-fork#305] [Q&A] [answered]
After using this project for training, the timbre transfer works smoothly, but occasionally there are some anomalies in pronunciation. I'm not sure if it's caused by the dataset of the pre-training model. I would like to try creating a pre-training model based on open source datasets. How can I do this?

**Comment by 34j** [comment, marked as answer]
You can just (adjust the hyperparameters and) start training from scratch. When you have finished your training, we would love to have you provide us with your model. (to save money and co2)
